acis_thermal_check/test_utils.py

```diff
@@ -1,4 +1,3 @@
-from ..utils import TASK_DATA
```

Comment:
I changed the name anyway just in case we end up putting something in this file that would be discovered in the future.